src/librustdoc/passes/stripper.rs

```diff
@@ -151,6 +153,23 @@ pub(crate) struct ImplStripper<'a> {
     pub(crate) document_private: bool,
 }
 
+impl<'a> ImplStripper<'a> {
+    #[inline]
```

Comment:
Why is this needed? Is their a policy for when to use #[inline], esp given this isn't used across crates

Reply:
It's just a habit of mine: when a function is used often and less than 15 lines, I put this attribute.